docs/event_channels.md

```diff
@@ -58,6 +58,7 @@ channel. This limit is increased to **25** if there is an active Premium members
 ## Purge non event messages
 
 By default non event messages (that aren't pinned) are purged from an event
-channel when a new event is posted in the channel.
+channel when a new event is posted or deleted. The goal of this feature is to
+maintain an event channel as a place for events only, free of distraction.
```

Comment:
One thing to consider is "how will this affect threads?" 

Would purge also archive/expire any old event threads? Would we just be letting those expire as per the user's server's rules?

Reply:
Purge has no effect on threads directly. If we delete a message that's threaded, the thread will live on behind the scenes, we just won't see a parent message any longer.

I'm not sure if this is worth documenting. Thoughts?

Reply:
Though that reminds me if an _event_ specifically is deleted, and the thread was created by Apollo, we do manually archive the thread right away to clean up.

That might be worth documenting under event deletion.

Reply:
To cover our bases, I think we'd benefit by telling the user what will or won't happen to threads, even if it's the default Discord behavior. Threads are still new and changing as evidenced by Discord's upcoming permission changes for Threads. I think we can expect they're probably going to make bot makers miserable by revising Threads again once they've collected some more user feedback.